src/Input/InputWrapper.tsx

```diff
@@ -0,0 +1,132 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import React from "react";
+import * as typography from "../typography";
+import { colors } from "../colors";
+import { IconAlertSolid } from "../icons/IconAlertSolid";
+
+interface Props {
+  /**
+   * Visible description
+   */
+  description?: React.ReactNode;
+
+  /**
+   * Defaults to standard
+   */
+  size: "small" | "standard" | "large";
+
+  /**
+   * Visible title
+   */
+  label?: React.ReactNode;
+
+  /**
+   * If true, the helperText is red with error icon
+   */
+  error?: boolean;
+
+  /**
+   * Displayed below the input
+   */
+  helperText?: React.ReactNode;
+
+  /**
+   * Displayed to the left of the helperText, an info icon for example
+   */
+  helperIcon?: React.ReactNode;
+
+  /**
+   * an icon to have at the right of the input box, a search for example
+   */
+  icon?: React.ReactNode;
+
+  /**
+   * generally a Stepper or an Input component
+   */
+  children: React.ReactNode;
+}
+
+export function InputWrapper({
+  label,
+  description,
+  size = "standard",
+  helperText,
+  helperIcon,
+  icon,
+  error,
+  children,
+  ...otherProps
+}: Props): ReturnType<React.FC> {
+  return (
+    <div {...otherProps}>
+      <label
+        css={{
+          paddingBottom: 8,
```

Comment:
Imo in a component library, a components box should be no larger than it needs to be, padding/margins should be set by the people using the components. However, this is a lot less annoying once https://github.com/mdg-private/engine-frontend/pull/1577 or something analogous lands so 🤷‍♂

Reply:
@jgzuke From our Space Kit meetings, it seems that all apollo products want the same spacing / font etc and that is why we decided to create a component library, to be able to use components that already have padding / margins etc and have builtin descriptions / error states.

Reply:
@mayakoneval the designs in https://app.zeplin.io/project/5d375cd742170577784b28ca/screen/5d41d60764e8c22bd5aeac20 have a smaller padding between them than two stacked inputs (with no labels)